patches/crypto-signature_verifier.cc.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/crypto/signature_verifier.cc b/crypto/signature_verifier.cc
+index 542c7f671b835d7b929a0623eca563ed8231fe3a..8f2cd17f29781281db695d1f54f174bc2535ad0c 100644
+--- a/crypto/signature_verifier.cc
++++ b/crypto/signature_verifier.cc
+@@ -12,6 +12,7 @@
+ #include "third_party/boringssl/src/include/openssl/digest.h"
+ #include "third_party/boringssl/src/include/openssl/evp.h"
+ #include "third_party/boringssl/src/include/openssl/rsa.h"
++#include "third_party/boringssl/src/include/openssl/err.h"
+ 
+ namespace crypto {
+ 
+@@ -44,6 +45,10 @@ bool SignatureVerifier::VerifyInit(SignatureAlgorithm signature_algorithm,
+       pkey_type = EVP_PKEY_EC;
+       digest = EVP_sha256();
+       break;
++    case ECDSA_SHA384:
```

Comment:
this patch doesn't follow patching standards, it should be `BRAVE_VERIFY_INIT` where `BRAVE_VERIFY_INIT` is defined in chromium_src, but in this case it can be eliminate with `define ECDSA_SHA256 ECDSA_SHA384 .... ECDSA_SHA384`